CMakeLists.txt

```diff
@@ -178,6 +177,29 @@ else(RDK_BUILD_PYTHON_WRAPPERS)
   find_package(Boost 1.39.0 REQUIRED)
 endif(RDK_BUILD_PYTHON_WRAPPERS)
 
+if(APPLE)
```

Comment:
On APPLE this checks to see if boost links to libstdc++ or libc++, if libc++ we also turn on c++11 to help future proof (unlike stdlibc++ libc++ is compatible between c++98 and c++11)